pydrad/parse/tests/test_strand.py

```diff
@@ -213,3 +223,18 @@ def test_ine_results(strand):
                 assert hasattr(profile, f'{element}_{i_z}')
                 ion_frac = getattr(profile, f'{element}_{i_z}')
                 assert ion_frac.shape == profile.coordinate.shape
+
+
+@pytest.mark.parametrize('property',[
+        'velocity',
+        'hydrogen_density',
+        'electron_density',
+        'electron_pressure',
+        'hydrogen_pressure',
+        'electron_temperature',
+        'hydrogen_temperature',
+])
+def test_phy_amr_consistency(property, strand_only_amr, strand_phy):
```

Comment:
One other possible test: before any heating, $T_{e} = T_{H}$.  Maybe it's worth checking consistency there as well?  Don't know if it adds anything, since this test covers the intended functionality in any case.

Reply:
Good point. I added a separate test for this. If we had had this in the first place, this also would have caught this bug!